addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -448,8 +450,14 @@ export class Wysiwyg extends Component {
             getPowerboxElement: () => {
                 const selection = (this.options.document || document).getSelection();
                 if (selection.isCollapsed && selection.rangeCount) {
-                    const baseNode = closestElement(selection.anchorNode, 'P:not([t-field]), DIV:not([t-field]):not(.o_not_editable)');
-                    const fieldContainer = closestElement(selection.anchorNode, '[data-oe-field]');
+                    const deepestNode = getDeepestPosition(selection.anchorNode, selection.anchorOffset);
+                    const elementSelectors = [
+                        'LI:not([t-field])',
+                        'DIV:not([t-field]):not(.o_not_editable)'
+                    ];
+                    elementSelectors.push(...paragraphRelatedElements.map(element => `${element}:not([t-field])`));
```

Comment:
Sorry, I did not notice this part of the code before. It would be better to write it this way:
```js
                    const elementSelectors = [
                        'LI:not([t-field])',
                        'DIV:not([t-field]):not(.o_not_editable)',
                        ...paragraphRelatedElements.map(tag => `${tag}:not([t-field])`),
                    ];
```